docs/flow/reference/components/prometheus.scrape.md

```diff
@@ -18,56 +18,40 @@ different labels.
 The following example will set up the job with certain attributes (scrape
 intervals, query parameters) and let it scrape two instances of the blackbox
 exporter. The received metrics will be sent over to the provided list of
-remote_writes, as defined by other components.
+receivers, as defined by other components.
 
 ```river
 prometheus.scrape "blackbox_scraper" {
-  targets = [
-    {"__address__" = "blackbox-exporter:9115", "instance" = "one"},
-    {"__address__" = "blackbox-exporter:9116", "instance" = "two"},
-  ]
-  forward_to = [prometheus.remote_write.grafanacloud.receiver, prometheus.remote_write.onprem.receiver]
-
-  scrape_config {
-    job_name        = "grafana"
-    scrape_interval = "10s"
-    params          = { "target" = ["grafana.com"], "module" = ["http_2xx"]}
-    metrics_path    = "/probe"
-  }
+	targets = [
+		{"__address__" = "blackbox-exporter:9115", "instance" = "one"},
+		{"__address__" = "blackbox-exporter:9116", "instance" = "two"},
+	]
+	forward_to = [prometheus.remote_write.grafanacloud.receiver, prometheus.remote_write.onprem.receiver]
+	
+	scrape_interval = "10s"
+	params          = { "target" = ["grafana.com"], "module" = ["http_2xx"] }
+	metrics_path    = "/probe"
 }
 ```
 
 ## Arguments
 
+The component will configure and start a new scrape job to scrape all of the
+input targets. The list of arguments that can be used to configure the block is
+presented below.
+
+Any omitted fields will take on their default values. In case that conflicting
+attributes are being passed (eg. defining both a BearerToken and
+BearerTokenFile or configuring both Basic Authorization and OAuth2 at the same
+time), the component will report an error.
```

Comment:
Do we define the behavior here? I assume error means that it will not work, but what if its an update? Does it go with last good configuration (like local.file) or does it fault entirely?

Reply:
Not sure; probably legit but maybe it should be referenced as part of the lifecycle pipeline. LGTM